source/guide_privatebin.rst

```diff
@@ -97,27 +99,27 @@ PrivateBin provides a .htaccess file, which blocks some known robots and link-sc
 
 Moving files outside of DocumentRoot
 ------------------------------------
-It is recommended_ to move the configuration, data files, templates and PHP libraries outside of your document root.
-To do that, create a folder  ``privatebin-data`` in ``/var/www/virtual/isabell/`` and move the folders to the new location (remember to replace ``isabell`` with your own username!).
+It is recommended_ to move the configuration, data files, templates and PHP libraries outside of your document root. This is useful to secure your installation.
+To do that, create a folder  ``privatebin-data`` in ``/home/isabell/`` and move the folders to the new location (remember to replace ``isabell`` with your own username!).
 If not already there, go to the ``html`` directory before running ``mv``.
 
 .. code-block:: console
 
  [isabell@stardust ~]$ cd ~/html
- [isabell@stardust html]$ mkdir /var/www/virtual/isabell/privatebin-data
- [isabell@stardust html]$ mv -t /var/www/virtual/isabell/privatebin-data cfg/ lib/ tpl/ vendor/
+ [isabell@stardust html]$ mkdir /home/$USER/privatebin-data
```

Comment:
you can use `$HOME` or `~` instead of `/home/$USER`

Reply:
done